html/elements/legend.json

```diff
@@ -24,29 +24,123 @@
               "version_added": "6"
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "3"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "1"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "1"
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "align": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": "18"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "1"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "≤6"
+              },
+              "opera": {
+                "version_added": "≤12.1"
+              },
+              "opera_android": {
+                "version_added": "≤12.1"
+              },
+              "safari": {
+                "version_added": "≤4"
+              },
+              "safari_ios": {
+                "version_added": "≤3"
+              },
+              "samsunginternet_android": {
+                "version_added": "1.0"
+              },
+              "webview_android": {
+                "version_added": "1"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "form": {
```

Comment:
So it looks like `<legend form>` isn't a content attribute. Try validating this markup:
```html
<!DOCTYPE html>
<html>
<head>
<title>Test</title>
</head>
<body>
<p></p>
<form id=bla></form>
<fieldset form=bla>
<select form=bla></select>
<label form=bla></label>
</fieldset>
</body>
</html>
```

The form attribute isn't allowed on `<legend>`. The spec is complicated, but from a quick read I think it makes sense:

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#form-owner